src/components/Header/Navigation.js

```diff
@@ -0,0 +1,96 @@
+import React, { useContext } from 'react';
+import firebase from 'firebase/app';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import colors from '../../styles/colors';
+import { Nav, NavItem, NavLink } from 'reactstrap';
+import { CityContext } from '../CityProvider/CityProvider';
+
+const StyledLink = styled(NavLink)`
+  text-transform: uppercase;
+  font-weight: bold;
+  font-size: 1rem;
+  color: ${colors.grey500} !important;
+
+  .nav-item:not(:last-child) & {
+    margin-right: 15px;
+  }
+
+  &.active {
+    color: ${colors.purple} !important;
+    border-bottom: 4px solid ${colors.purple} !important;
+  }
+
+  &:hover {
+    color: ${colors.grey500} !important;
+  }
+`;
+
+const ExternalLink = styled(NavLink)`
+  text-transform: uppercase;
+  font-weight: bold;
+  font-size: 1rem;
+  color: ${colors.grey500} !important;
+
+  .nav-item:not(:last-child) & {
+    margin-right: 15px;
+  }
+
+  &.active {
+    color: ${({ theme }) => theme.primaryColor} !important;
+  }
+
+  &:hover {
+    color: ${({ theme }) => theme.primaryColor} !important;
+    text-decoration: none;
+  }
+`;
+
+const Navigation = () => {
+  const handleActive = () => {
+    return true;
+  };
+
+  const { cityPath } = useContext(CityContext);
+
+  return (
+    <Nav className="mr-0 ml-auto" navbar>
+      <NavItem>
+        <StyledLink
+          tag={Link}
+          to={`${cityPath}/questionario`}
+          onClick={handleActive}
+          active={handleActive ? true : false}
```

Comment:
não entendi esse ternário, se a função `handleActive` só retorna um `true`, o ternario vai ser sempre `true`

```suggestion
          active={true}
```

Reply:
mas se colocar active={true} ele não vai estar sempre ativo?
eu não sabia em qual opção trocar pelo activeClassName que tava no site